lighthouse/tests/beacon_node.rs

```diff
@@ -206,7 +206,35 @@ fn eth1_purge_cache_flag() {
         .with_config(|config| assert!(config.eth1.purge_cache));
 }
 
-// Tests for Merge flags.
+// Tests for Bellatrix flags.
+#[test]
+fn merge_flag() {
+    CommandLineTest::new()
+        .flag("merge", None)
+        .run_with_zero_port()
+        .with_config(|config| assert!(config.execution_endpoints.is_some()));
+}
+#[test]
+fn merge_execution_endpoints_flag() {
+    use sensitive_url::SensitiveUrl;
+    let urls = vec!["http://sigp.io/no-way:1337", "http://infura.not_real:4242"];
+    let endpoints = urls
+        .iter()
+        .map(|s| SensitiveUrl::parse(s).unwrap())
+        .collect::<Vec<_>>();
+    let mut endpoint_arg = urls[0].to_string();
+    for url in urls.into_iter().skip(1) {
+        endpoint_arg.push(',');
+        endpoint_arg.push_str(url);
+    }
```

Comment:
I might be misunderstanding, but you can probably just do:
```rust
let endpoint_arg = urls.join(",");
```

Reply:
@macladson agreed.. this is how I had tried to write it initially. But `join` isn't a function. What we need is [`intersperse`](https://doc.rust-lang.org/nightly/std/iter/struct.Intersperse.html), which is currently nightly-only :/

When that becomes available, this code nicely simplifies to just:
`let endpoint_arg = urls.into_iter().intersperse(',').collect::<String>();`

I tried opting into using intersperse, but our CI tests prevent me from committing that for release tests :/

Reply:
For a `Vec` of `&str`'s (or `String`'s) you can use `join`: https://doc.rust-lang.org/std/vec/struct.Vec.html#method.join

[Playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=c0af1c50463bc12f12a47a3e6aee3b7a)

`join` is also used [elsewhere in the tests](https://github.com/sigp/lighthouse/blob/unstable/lighthouse/tests/validator_client.rs#L147).

I could be wrong, but I think that should work here.